rainbow-me/browser-extension
# [BX-213][BX-214][BX-209] L2 TX + Add TX Handler + Add Pending TX on send

## What changed (plus any additional context for devs)
- l2 transactions + l2 pending tx resolution
- add tx handler
- add pending tx after send

## Screen recordings / screenshots
pending tx example: https://recordit.co/t2II0IAK93
L2 activity example: https://recordit.co/SEDbuNclh2


## What to test
- ensure pending transactions appear after initiating send and they resolve correctly
- ensure l2 activity appears in activity section

## Final checklist
